include/lsst/afw/geom/SpanSet.h

```diff
@@ -216,6 +217,12 @@ class SpanSet : public afw::table::io::PersistableFacade<lsst::afw::geom::SpanSe
      */
     std::shared_ptr<SpanSet> transformedBy(XYTransform const &t) const;
```

Comment:
Is the XYTransform version still being kept?

Reply:
Yes. It is likely that we can remove the XYTransform versions of most or all code once this ticket is merged, but that should be a different ticket and likely an RFC as well.